packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -595,6 +620,53 @@ export const RemoteServerFactory = {
         })
     },
 
+    /**
+     * @private
+     */
+    _setupSlasPrivateClientProxy(app, options) {
+        if (!options.useSLASPrivateClient) {
+            return
+        }
+        const endpointPath = '/mobify/scapi/shopper/auth'
```

Comment:
[NIT] Would it make sense to pull this out into `constants.js`?

Reply:
This is a good idea. I will do it in a follow up PR. I'd like to get this merged in since #1676 builds on top of this.

Reply:
The endpointPath has been moved to `constants.js` in #1679